cuda_core/tests/memory_ipc/conftest.py

```diff
@@ -0,0 +1,32 @@
+# SPDX-FileCopyrightText: Copyright (c) 2025 NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+import pytest
+from cuda.core.experimental import Device, DeviceMemoryResource
```

Comment:
Would be nice to get `DeviceMemoryResourceOptions` exercised
```suggestion
from cuda.core.experimental import Device, DeviceMemoryResource, DeviceMemoryResourceOptions
```